src/js/containers/list-page.js

```diff
@@ -32,22 +33,24 @@ class ListPage extends Component {
       margin: '5px 10px 0px 5px'
     } ];
     return (
-      <div className="page-content">
-        <RaisedButton style={ buttonStyles[ 0 ] }
-          label="Map"
-          labelPosition="after"
-          primary={ true }
-          icon={ <MapIcon /> }
-          onClick={ ( ) => history.push( '/' ) } />
-        <RaisedButton style={ buttonStyles[ 1 ] }
-          label="Filter"
-          labelPosition="after"
-          primary={ true }
-          icon={ <FilterIcon /> }
-          onClick={ ( ) => history.push( 'filter' ) } />
-        <PointList points={ this.props.services }
-          onPointClick={ this.onPointClick.bind( this ) } />
-      </div>
+      <Page className="layout__section">
+        <Paper>
```

Comment:
Can we leave out the Paper here? Later, we should look into how the component looks on desktop and mobile, but for just mobile, it's ok to have the list directly "flat" on on the screen. This will also get rid of the "double margins" in the screenshot.